catalog-info.yml

```diff
@@ -0,0 +1,28 @@
+---
+# GitLab CI/CD Catalog metadata for RADIUSS Shared CI
+spec:
+  inputs:
+    # This catalog provides reusable components for RADIUSS projects
+    # See README.md for usage examples
+  components:
+    machine-pipeline:
```

Comment:
maybe clearer to just list all machine pipelines like dane-pipleline, corona-pipeline etc